pod-provider/frontend/src/config/messages/fr.js

```diff
@@ -193,6 +193,7 @@ module.exports = {
     },
     message: {
       copied_to_clipboard: 'Copié !',
+      creating_invite_link: 'Créer...',
```

Comment:
In english there are two other messages 
```suggestion
      loading_invite_link: 'Chargement en cours...',
      loading_invite_link_failed: 'Chargement échoué',
```